cmd/jobsClean.go

```
@@ -30,13 +30,44 @@ import (
 	"github.com/Azure/azure-storage-azcopy/v10/common"
 )
 
+type JobsCleanOptions struct {
+	WithStatus common.JobStatus
+}
+
+func RunJobsClean(opts JobsCleanOptions) error {
+	err := handleCleanJobsCommand(opts.WithStatus)
+	if err == nil {
+		if opts.WithStatus == common.EJobStatus.All() {
+			glcm.Exit(func(format common.OutputFormat) string {
+				return "Successfully removed all jobs."
+			}, common.EExitCode.Success())
+		} else {
+			glcm.Exit(func(format common.OutputFormat) string {
+				return fmt.Sprintf("Successfully removed jobs with status: %s.", opts.WithStatus)
+			}, common.EExitCode.Success())
+		}
+	} else {
+		glcm.Error(fmt.Sprintf("Failed to remove log/plan files due to error: %s.", err))
```

Should we return an error from here?

---

same comment as below.